plugins/web-search/web-search.plugin.zsh

```diff
@@ -25,6 +25,7 @@ function web_search() {
     archive         "https://web.archive.org/web/*/"
     scholar         "https://scholar.google.com/scholar?q="
     ask             "https://www.ask.com/web?q="
+    youtube         "https://www.youtube.com/results?search_query="
```

Comment:
Does https://github.com/ohmyzsh/ohmyzsh/blob/master/plugins/web-search/README.md needs to be updated too?

Reply:
It should be, yes!

Reply:
Fixed in c91684bb1b723833a96415c2c724332a6e2e620f
Thanks for the report!